circle.yml

```diff
@@ -45,6 +45,7 @@ jobs:
             if [ "$CIRCLE_NODE_INDEX" != "3" ]; then exit; fi
             dockerfile=dockerfiles/Dockerfile.dev
             echo "COPY . ." >> $dockerfile
+            rm -f .dockerignore # include .git
```

Comment:
Ya, I don't know of a better option without https://github.com/moby/moby/issues/12886